### ISSUE TITLE
Episodes greater than 1000 being treated as SSEE

### ISSUE THREAD

**Issue by Winter** [scttcper/video-filename-parser#65]
When parsing something like `A Random Series Name - 1000 [1080p WEB-DL]` it will register it as Season 10, Episode 0. 

Obviously there's the glaring issue of the SSEE being broken with any attempt to rectify so maybe an optional field can be added called "absoluteEpisodeNumber" if there's any ambiguity?

**Comment by scttcper** [comment]
E1000 is handled, but i don't know if i'll ever handle just `title - 1000` its pretty vague

**Comment by Winter** [comment]
Yeah, I can understand that. But every Anime release name uses absolute episodes and take One Piece for example which has just hit the 1000 episode mark. There are others that meet this criteria such as Pokemon and Detective Conan. 

Quite frankly I've never in all my years seen anything use SSEE.

Thanks for adding EXXXX though!

**Comment by scttcper** [comment]
I saw `One.Piece.E1001.SUBFRENCH.1080p.WEB.x264-AMB3R` which this should parse correctly. 

I'll try adding something for `One Piece - 1001 [1080p]`

**Comment by Winter** [comment]
Thanks - `EXXXX` actually breaks release name rules on a lot of private "bluray" enthusiast sites. My only concern is breaking the SSXX format, even though I've never seen it
